core/src/main/java/io/questdb/cutlass/http/HttpConstants.java

```diff
@@ -69,6 +69,7 @@ public final class HttpConstants {
     public static final Utf8String URL_PARAM_STATUS_TABLE_NAME = new Utf8String("j");
     public static final Utf8String URL_PARAM_TIMESTAMP = new Utf8String("timestamp");
     public static final Utf8String URL_PARAM_TIMINGS = new Utf8String("timings");
+    public static final Utf8String URL_PARAM_CREATE_TABLE = new Utf8String("createTable");
```

Comment:
nit: I'd call it `create` since we already have `overwrite`, not `overwriteTable`